File: 2502-sort-the-people/sort-the-people.py
class Solution(object):
    def sortPeople(self, names, heights):
        """
        :type names: List[str]
        :type heights: List[int]
        :rtype: List[str]
        """
        sorted_names = [x for _, x in sorted(zip(heights, names), reverse=True)]
        return sorted_names

File: 2502-sort-the-people/sort-the-people.js
/**
 * @param {string[]} names
 * @param {number[]} heights
 * @return {string[]}
 */
var sortPeople = function(names, heights) {
    let combined = names.map((name, index) => ({ name: name, height: heights[index] }));
    combined.sort((a, b) => b.height - a.height);
    let sorted_names = combined.map(person => person.name);

    return sorted_names;
};